inspire_json_merger/merger_config.py

```diff
@@ -80,11 +80,22 @@ def factory(config_type):
 
 
 class MergerConfigurationOperations(object):
+    """General Configuration for the Merger
+
+    Attributes:
+        default_dict_merge_op   Default operation for the merge of dicts
+        default_list_merge_op   Default operation for the merge of lists
+        list_dict_ops = None    Dictionary of rules for the merge of dicts
+        list_merge_ops = None   Dictionary of rules for the merge of dicts
+        comparators = None      Dictionary of comparators
+        conflicts = None        List of conflicts that have to be filtered
+    """
     default_dict_merge_op = DictMergerOps.FALLBACK_KEEP_UPDATE
     default_list_merge_op = UnifierOps.KEEP_ONLY_UPDATE_ENTITIES
     list_dict_ops = None
     list_merge_ops = None
     comparators = None
+    conflicts = None
```

Comment:
The name of this var is not so self explaining...what do you think about something like `relevant_conflicts` or `conflicts_to_show`?